lib/iex/lib/iex/helpers.ex

```diff
@@ -393,14 +393,19 @@ defmodule IEx.Helpers do
   end
 
   @doc """
-  Prints information about the data type of the given term.
+  Prints information about the data type of any vaild Elixir term.
```

Comment:
I'm not sure "invalid Elixir term" is a thing: every Elixir term is valid for the intrinsic reason of it being an Elixir term 😕